### ISSUE TITLE
@mantine/core - Select - issue with defaultValue when using Placeholder

### ISSUE THREAD

**Issue by cstrat** [mantinedev/mantine#99]
https://codesandbox.io/s/mantine-select-error-mkurq

```
Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.
```

This warning only appears when using `placeholder`.
See in the codesandbox, if you comment out the placeholder the warning doesn't appear.

**Comment by rtivital** [comment]
Seems like an easy thing to fix, do you want to create a PR for this? If not, I'll resolve it on my own in next minor release

**Comment by cstrat** [comment]
I'll take a look to see if I can provide a fix!

What would the best approach be though?
We need to assign a `value` to the placeholder option and then have that as the `defaultValue` on the parent select box.

Assign a random/arbitrary value? How far do you want to go - do we check to see if that value is in use amongst the options?

**Comment by rtivital** [comment]
Resolved in [1.3.0](https://github.com/mantinedev/mantine/releases/tag/1.3.0)
